Makefile

```diff
@@ -0,0 +1,29 @@
+DESTDIR = /usr/local
+BINDIR = $(DESTDIR)/bin
+MANDIR = $(DESTDIR)/share/man
+
+paru:
+	@set -o errexit -o pipefail
```

Comment:
This is not needed, as make will exit if any of the commands fail anyway, and there are no pipes for pipefail to affect. Additionally, pipefail is bash-specific and may not work if the shell being used by make is not bash. Finally, depending on implementation, each command in the target might run in its own shell, so the set call might not affect the following calls at all.